components/brave_referrals/browser/brave_referrals_service.h

```diff
@@ -49,8 +51,6 @@ class BraveReferralsService {
   std::string BuildReferralFinalizationCheckPayload() const;
   void FetchReferralHeaders();
   void CheckForReferralFinalization();
-  std::string FormatExtraHeaders(const base::Value* referral_headers,
-                                 const GURL& url);
 
   // Invoked from RepeatingTimer when referral headers timer fires.
   void OnFetchReferralHeadersTimerFired();
```

Comment:
We shouldn't need `OnFetchReferralHeadersTimerFired` now, right?

Reply:
Removed.